### ISSUE TITLE
Mixed decklists with the opponent and I could see a card he drew

### ISSUE THREAD

**Issue by kazumaverdao** [HearthSim/Hearthstone-Deck-Tracker#3261]
### Expected Behavior
On the right I have my list, on the left, the cards the opponent has already played.

### Actual Behavior
His cards were showing on the right in between my cards, which were all "solid color", as like I never drew any card. When I drew cards, they were being actually counted on the left. I have used this template for months. I did not change the position of the lists in the settings. 

As it was bugged, I wasn't really paying attention to the tracker, but on his last turn, the moment he drew the start of the turn card, for some reason I paid attention to the tracker, and I was able to see The Storm Guardian leaving the right-side list. And he played exactly the last card of his hand (the one we just drew), and it was, in fact, The Storm Guardian. Even though it didn't made any difference (I won on the next turn), I was kind of able to cheat seeing his draw.

![hdt bug](https://cloud.githubusercontent.com/assets/17756495/26513517/e79cf35a-4239-11e7-9f7a-72d32056b597.jpg)

All the dark cards on the right side were played by my opponent. The "not-dark" cards in the left list were played by me. I guess the dark cards on the left are the ones I mulliged. Also, on the top of the left-side list, it says I'm playing against Rogue, when in fact it was Shaman. 

### Steps to reproduce behavior
I don't really know how I did it. I restarted everything and it's ok now. I'm just reporting because I saw the card he drew.

### Log/Screenshots
[hdt_log_1495833703.txt](https://github.com/HearthSim/Hearthstone-Deck-Tracker/files/1033016/hdt_log_1495833703.txt)

My name only appears once in the log:
5:05:13 PM|Info|GameV2.CacheMatchInfo >> Kazuma vs gmangel
and the opponent name doesn't match the one I played against.

**Comment by kazumaverdao** [comment]
It just started to happen again.

![2 hdt bug](https://cloud.githubusercontent.com/assets/17756495/26513831/b139847a-423b-11e7-8b70-87c500e57f04.jpg)

I guess it is related to Swashburglar. About the original post, I think I saw The Storm Guardian because when I killed Blue Eyes and The Storm Guardian was sent to his deck, it probably went to the righ-side list, as it was to my deck. I wasn't really paying attention.
